### ISSUE TITLE
Use path in base URL

### ISSUE THREAD

**Issue by adrianke** [tsileo/microblog.pub#47]
I see that there is a base URL assembled by values from the config, but there aren't any option for specifying a path.

The use case is for existing websites. One would have to proxy some files in `/. well-known` apart from the specified path in order to make this work.

Would this use case be of interest?

**Comment by tsileo** [comment]
Hey, as the server comes with a front-end, I haven't really thought about it (not sure how easy it would be to do it), so for now this is definitely not a focus.

I will leave the issue open in case I have some time to get back to this later.

**Comment by sentriz** [comment]
yeah would be cool to be `me@me.com` on the fediverse but host microblog elsewhere eg. `microblog.me.com`
i tried 
```diff
diff --git a/config.py b/config.py
index 2d278dd..0814535 100644
--- a/config.py
+++ b/config.py
@@ -63,7 +63,7 @@ with open(os.path.join(KEY_DIR, "me.yml")) as f:
     NAME = conf["name"]
     DOMAIN = conf["domain"]
     SCHEME = "https" if conf.get("https", True) else "http"
-    BASE_URL = SCHEME + "://" + DOMAIN
+    BASE_URL = conf["base_url"]
     ID = BASE_URL
     SUMMARY = conf["summary"]
     ICON_URL = conf["icon_url"]
```
and updated my config
```
domain: 'me.com'
username: 'me'
icon_url: 'https://microblog.me.com/static/nopic.png'
base_url: 'https://microblog.me.com'
```

and also am routing `me.com/.well-known/webfinger` -> `microblog.me.com/.well-known/webfinger`
and the curl looks good `curl -L "https://me.com/.well-known/webfinger?resource=acct:me@me.com"`

but can't find myself on mastodon with the `me@me.com` name. maybe something key related? or is there more to this than I thought :)

**Comment by adrianke** [comment]
@sentriz According to this ( https://blog.joinmastodon.org/2018/06/how-to-implement-a-basic-activitypub-server/ ), you have done what is required. Not sure why it fails.

**Comment by sentriz** [comment]
hmm me neither, ill try again some other time
